internal/pkg/scaffold/olm-catalog/csv.go

```diff
@@ -79,14 +79,14 @@ func (s *CSV) GetInput() (input.Input, error) {
 		return input.Input{}, ErrNoCSVVersion
 	}
 	if s.Path == "" {
-		lowerProjName := strings.ToLower(s.ProjectName)
+		operatorName := strings.ToLower(s.OperatorName)
```

Comment:
No need to use `strings.ToLower()` here since the user is specifying exactly what name they want for their operator.

Reply:
the `--operator-name` flag is optional. If it is empty `ProjectName` (project base directory) is used.
It might need formatting.
Hence, wouldn't it be better to keep `strings.ToLower()`

Reply:
I would perform that operation [here](https://github.com/operator-framework/operator-sdk/pull/1571/files#diff-b1ad7fe7bb19383ad91bba7bde3d16c1R88) instead.

Reply:
Looks like by convention the operator name should be lowercased.